docs/gitlab/02-activation.md

```diff
@@ -107,11 +117,3 @@ All you need is [docker](https://www.docker.com/) installed on your machine.
 6. Copy the content to your CI's environment variable `UNITY_LICENSE`.
    _Note: if you are doing this on windows, chances are the [line endings will be wrong as explained here](https://gitlab.com/gableroux/unity3d-gitlab-ci-example/issues/5#note_95831816). Luckily for you, [`.gitlab-ci.yml`](https://gitlab.com/gableroux/unity3d-gitlab-ci-example/-/blob/master/.gitlab-ci.yml) solves this by removing `\r` character from the env variable so you'll be alright_
    [`.gitlab-ci.yml`](https://gitlab.com/gableroux/unity3d-gitlab-ci-example/-/blob/master/.gitlab-ci.yml) will then place the `UNITY_LICENSE` to the right place before running tests or creating the builds.
-
-## Unity license per target
```

Comment:
This was true for gableroux/unity3d images, but it doesn't make sense anymore with `unityci/editor` 👍